### DISCUSSION TITLE
How to get the "syllabus_body" for a specific course?

### DISCUSSION THREAD

**Discussion by yfmichea** [ucfopen/canvasapi#644] [Q&A] [answered]
Hi!
I am looking for help to pull the syllabus for a course using canvasapi. Is there a get_syllabus() equivalent?
Thanks!

**Comment by bainco** [comment, marked as answer]
When you get the course (using `get_course`) you need to include the `syllabus_body` parameter:

https://canvas.instructure.com/doc/api/courses.html

And then the syllabus body is available via the `syllabus_body` property of the course.

```
canvas = Canvas(canvas_url, canvas_token)
the_course = canvas.get_course(course_id, include="syllabus_body")
print(the_course.syllabus_body)
```

**Reply by yfmichea** [reply]
Thanks for your help!
Yanko
